examples/simple_auth/simple_resource.py

```diff
@@ -1,15 +1,15 @@
+# coding=utf-8
+# pylint: disable=no-name-in-module, import-error, unused-argument
```

Comment:
I know you want to improve examples later but at least keep only the warnings that make sense. In most examples modules, you disabled too many of them once only one or two are valid 😉

Reply:
done